src/components/validation.js

```diff
@@ -0,0 +1,20 @@
+export default function validation(values) {
+    let errors = {};
```

Comment:
seems like not using this file as well

Reply:
leave it for later reusable class for validation